enhancements/api-review/custom-tags-aws.md

```diff
@@ -31,35 +37,51 @@ have the ability to apply user defined tags to many resources created by OpenShi
 Note: this enhancement is slightly retroactive. Work has already begun on this. See
 - https://github.com/openshift/api/pull/864
 - https://github.com/openshift/cluster-ingress-operator/pull/578
+- https://github.com/openshift/api/pull/1064/
 
 ## Motivation
 
 Motivations include but are not limited to:
 
-- allow admin, compliance, and security teams to keep track of assets and objects created by OpenShift,
+- Allow admin, compliance, and security teams to keep track of assets and objects created by OpenShift,
    both at install time and during continuous operation (Day 2)
-- in a Managed OpenShift environment such as Red Hat OpenShift on AWS (ROSA), allow easy differentiation
+- In a Managed OpenShift environment such as Red Hat OpenShift on AWS (ROSA), allow easy differentiation
    between Red Hat-managed AWS objects and customer-managed objects
-- *allow for the restriction of permissions granted to Red Hat in an AWS account by AWS resource tags.
+- *Allow for the restriction of permissions granted to Red Hat in an AWS account by AWS resource tags.
    For example, see https://issues.redhat.com/browse/SDE-1146 - "IAM users and roles can only operate on resources with specific tags"*
 
 ### Goals
 
-- the administrator or service (in the case of Managed OpenShift) installing OpenShift can pass an arbitrary
+- The administrator or service (in the case of Managed OpenShift) installing OpenShift can pass an arbitrary
    list of user-defined tags to the OpenShift Installer, and everything created by the installer and all other
    bootstrapped components will apply those tags to all resources created in AWS, for the life of the cluster, and where supported by AWS.
-- tags must be applied at creation time, in an atomic operation. It isn't acceptable to create an object and,
-   after some period of time, apply the tags post-creation.
+- User-defined tags can be applied at creation time, in an atomic operation.
```

Comment:
s/in an atomic operation/as an atomic operation/

Reply:
Thanks for the review, updated the EP accordingly.